.github/workflows/docker_compose.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker Image CI
```

Comment:
@oh6hay this file configures a github workflow that will build the docker containers on every PR push. 

You would have to turn it on in the "Actions" tab of this repo.
https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow